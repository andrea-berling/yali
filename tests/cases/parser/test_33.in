(94 != 30) == ((-85 + 67) >= (84 * 41))
