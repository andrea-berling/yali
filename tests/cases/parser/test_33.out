(== (group (!= 94.0 30.0)) (group (>= (group (+ (- 85.0) 67.0)) (group (* 84.0 41.0)))))
